cmd/jaeger/internal/extension/jaegerstaticsampling/README.md

```diff
@@ -0,0 +1,13 @@
+# jaeger_static_sampling
+
+This module implements an extension that offers a static sampling strategy store for Jaeger. It enables the configuration of sampling strategies from a local file and automatically reloads them at specified intervals.
```

Comment:
> offers a static sampling strategy

offers to whom? As a user how would I use it?

Reply:
I updated the README and listed it in depth, explaining the use case and how to configure it.